symphony-bdk-examples/bdk-spring-boot-example/src/main/java/com/symphony/bdk/examples/spring/GifFormActivity.java

```diff
@@ -37,6 +37,11 @@ public void displayGifForm(CommandContext context) {
     this.messageService.send(context.getStreamId(), message);
   }
 
+  @Slash("/echo {one} {two}")
+  public void echo(CommandContext context, String one, String two) {
```

Comment:
do we start introducing @SlashParam annotations too (similar to PathParam)? To customize the mappings or have optional parameters

Reply:
do we support multiple types here?

Reply:
No, not yet. But we could. See discussion in the confluence page.
I would say specifying the fact a param is optional should be put in the command pattern and should put after all mandatory params.